### ISSUE TITLE
Smooth Pinball Neural Network for Probabilistic Forecasting of Wind Power

### ISSUE THREAD

**Issue by yujiariyasu** [yujiariyasu/paper_summary#7]
# URL
https://arxiv.org/pdf/1710.01720.pdf

# Overview
This paper analyzes the effectiveness of nonparametric probabilistic forecasting of wind power that combines a smooth approximation of the pinball loss function with a neural network.

# Point
